cla_public/static-src/stylesheets/_sidebar.scss

```diff
@@ -8,10 +8,14 @@ body.cla-dev .sidebar {
   border-top: 10px solid $laa-dev-colour;
 }
 
-.sidebar {
+body .sidebar {
```

Comment:
I'm not sure what difference this would make.. is this to increase specificity?

Reply:
It overrides CSS elsewhere.  Otherwise the coloured bar is not shewn.
![image](https://user-images.githubusercontent.com/32877315/94918611-d33eed00-04aa-11eb-90dc-282757d25fbe.png)